pipeline/transformer/transformer_sierra/src/test/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraMergeCandidatesTest.scala

```diff
@@ -260,21 +269,25 @@ class SierraMergeCandidatesTest
     }
   }
 
-  private def createMiroPictureWithUrls(urls: List[String]): SierraBibData =
-    createMiroPictureWith(varFields = create962subfieldsWith(urls = urls))
+  private def createPictureWithUrls(urls: List[String]): SierraBibData =
+    createPictureWith(varFields = create962subfieldsWith(urls = urls))
 
-  private def createMiroPictureWith(varFields: List[VarField]): SierraBibData =
-    createSierraBibDataWith(
-      materialType = Some(SierraMaterialType(code = "k")),
-      varFields = varFields
-    )
+  private def createPictureWith(varFields: List[VarField]): SierraBibData =
+    createBibDataWith(varFields = varFields, materialTypeCode = 'q')
+
+  private def createDigitalImageWith(varFields: List[VarField]): SierraBibData =
+    createBibDataWith(varFields = varFields, materialTypeCode = 'q')
 
-  private def createMiroDigitalImageWith(
-    varFields: List[VarField]): SierraBibData =
+  private def create3DObjectWith(varFields: List[VarField]): SierraBibData =
+    createBibDataWith(varFields = varFields, materialTypeCode = 'r')
+
+  private def createBibDataWith(varFields: List[VarField],
+                                materialTypeCode: Char) = {
     createSierraBibDataWith(
-      materialType = Some(SierraMaterialType(code = "q")),
+      materialType = Some(SierraMaterialType(code = materialTypeCode.toString)),
```

Comment:
Complete nitpick - but why doesn't this just take a `char`?
[Looking at the list, we ran out of IDs at `z`](https://github.com/wellcometrust/platform/blob/master/ontologies/Reference%20data/sierra-material-types.csv).

Reply:
Agreed, personally I'd have types for all these.